infra/roles/products/files/restartApp.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
This is wrong as you won't have a systemd service, and need some other way to redeploy your application.